src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -32,6 +32,18 @@
                   Description="Specifies the base namespace for files added to the project."
                   Category="General" />
 
+  <BoolProperty Name="TargetMultipleFrameworks"
+                DisplayName="Target multiple frameworks"
+                Category="General">
```

Comment:
Bool properties should have a description. We use it as the label on the check box. Without this, the header is used instead and no header is visible for the property.

```suggestion
  <BoolProperty Name="TargetMultipleFrameworks"
                DisplayName="Target multiple frameworks"
                Description="Build this project for multiple target frameworks."
                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147236"
                Category="General">
```